services/121-service/src/payments/payments.controller.ts

```diff
@@ -77,7 +77,8 @@ export class PaymentsController {
     }[]
   > {
     // TODO: REFACTOR: use a DTO to define stable structure of result body
-    return await this.paymentsService.getPayments(programId);
+    const payments = await this.paymentsService.getPayments(programId);
+    return payments.sort((a, b) => a.payment - b.payment);
```

Comment:
I implemented the sorting in the BE instead of FE, but open to suggestions

Reply:
How were the payments sorted before this change?
Would it make sense to sort them by date (if possible) rather than by id?
Are you sure that you have adjusted/checked all the places this is used in the old portal?

Reply:
It wasn't sorted before, I now changed it to sort by date (directly in the query). Old portal still works as far as I can see